iglu-registry/0.3.0/scripts/super_api_key_init.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+pswd=$(grep password /snowplow/config/application.conf | cut -d '=' -f 2 | tr -d '"' | tr -d '\r')
+host=$(grep host /snowplow/config/application.conf | cut -d '=' -f 2 | tr -d '"' | tr -d '\r' | tail -n1)
+port=$(grep port /snowplow/config/application.conf | cut -d '=' -f 2 | tr -d '"' | tr -d '\r' | tail -n1)
+dbname=$(grep dbname /snowplow/config/application.conf | cut -d '=' -f 2 | tr -d '"' | tr -d '\r')
+username=$(grep username /snowplow/config/application.conf | cut -d '=' -f 2 | tr -d '"' | tr -d '\r')
+PGPASSWORD=$(echo $pswd) psql -h $(echo $host) -p $(echo $port) -d $(echo $dbname) -U $(echo $username) -c "insert into apikeys (uid, vendor_prefix, permission, createdat) values ($(echo "'"$API_KEY"'"), '.', 'super', current_timestamp);"
```

Comment:
since we're making the user launch this script, why not make the user directly launch the psql command?

Also, @jbeemster suggested that maybe we should directly specify the super api key as a secret in the config and the table bootstrapping would take care of inserting it directly in the table, which I think is a good idea.
We would need to find a way to store secrets in the config though.

Wdyt @chuwy @oguzhanunlu ?

Reply:
ah snowplow/iglu#63

Reply:
so could we remove this and make the user launch the psql command directly at least?

Reply:
Sure, [this command](https://github.com/snowplow/snowplow-docker/blob/iglu-registry/iglu-registry/README.md#insert-the-super-api-key) will be changed to something like `docker exec iglu-registry psql ...` instead of `docker exec iglu-registry sh ...`.